modules/tinymce/CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Whitespace between transparent elements would incorrectly be converted into empty paragraphs. #TINY-9761
 - Pressing arrow keys inside RTL elements would move the caret in an incorrect direction when moving over elements with the `contenteditable` attribute set to `false`. #TINY-9565
 - Inserting table consecutively without focus in the editor would result in the table being inserted at the wrong position. #TINY-3909
+- Pressing enter in `<blockquote><p><strong>&#xFEFF;</strong></p></blockquote>` create a newline instead of remove `blockquote`. #TINY-9794
```

Comment:
Maybe just skip some details, for example
```suggestion
- In some cases, the exiting a `blockquote` element could fail when the cursor was positioned at the end of the `blockquote`. #TINY-9794
```

Reply:
done